File: Python/Negative Image.py
from PIL import Image, ImageChops
image_path = r'Enter the path to the image here'

try:
    img = Image.open(image_path)
    if img.mode != 'RGB':
        img = img.convert('RGB')
    inv_img = ImageChops.invert(img)
    inv_img.show()
except Exception as e:
    print(f"An error occurred: {e}")

File: MATLAB/Negative Image.m

    img = imread('Enter the path to the image here');
    img = im2double(img)
    negativee = 1 - img
        negative_img = 255 - img;
  
    figure;
    subplot(1, 2, 1);
    imshow(img);
    title('Original Image');
    
    subplot(1, 2, 2);
    imshow(negativee);
    title('Negative Image')
